src/tests/BuildWasmApps/Wasm.Build.Tests/PInvokeTableGeneratorTests.cs

```diff
@@ -119,6 +119,198 @@ public static int Main()
             Assert.Contains("Main running", output);
         }
 
+        [Theory]
+        [BuildAndRun(host: RunHost.None)]
+        public void UnmanagedStructsAreNotConsideredBlittable(BuildArgs buildArgs, string id)
```

Comment:
The 3 tests seem to have lot of the same code. That can be moved to a separate method.